### PR TITLE
[APM][.NET] Release 2.28.0 changes

### PR DESCRIPTION
**Pull request by pierotibou**
### What does this PR do?
Adds new parameters introduced in 2.28.0

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/tracing/trace_collection/library_config/dotnet-core.md [hunk 1]
```diff
@@ -201,6 +201,17 @@ Note that the delimiter is a comma and a space: `, `.
 **Example**: `mysql:main-mysql-db, mongodb:offsite-mongodb-service`<br>
 The `from-key` value is specific to the integration type, and should exclude the application name prefix. For example, to rename `my-application-sql-server` to `main-db`, use `sql-server:main-db`. Added in version 1.23.0
 
+`DD_HTTP_SERVER_TAG_QUERY_STRING`
+: When set to `true` query string parameters are added to the http.url tag. You can find more details in [Redacting the query in the url][14].
```
Comment (andrewlock) [inline content/en/tracing/trace_collection/library_config/dotnet-core.md:205]:
```suggestion
: When set to `true`, the `http.url` includes query string parameters. You can find more details in [Redacting the query in the url][14].
```

File: content/en/tracing/trace_collection/library_config/dotnet-framework.md [hunk 2]
```diff
@@ -217,6 +217,16 @@ Added in version 1.17.0. <br>
...
+`DD_HTTP_SERVER_TAG_QUERY_STRING`
+: When set to `true` query string parameters are added to the http.url tag. You can find more details in [Redacting the query in the url][14].
+**Default**: `true`
+
+`DD_HTTP_SERVER_TAG_QUERY_STRING_SIZE`
+: When `DD_HTTP_SERVER_TAG_QUERY_STRING` is true, sets the max size of the querystring to report, before obfuscation. Set 0 for no limit in size<br>
+**Default**: `5000`
+
+`DD_TRACE_OBFUSCATION_QUERY_STRING_REGEXP`
+: When `DD_HTTP_SERVER_TAG_QUERY_STRING` is true, this regex redacts sensitive data from incoming requests' query string reported in the `http.url` tag (matches are replaced with `<redacted>`). An empty string disables redaction. This regex executes for every incoming request.
```
[the remark this answers is not in the document]
Comment (pierotibou) [reply content/en/tracing/trace_collection/library_config/dotnet-framework.md:229]: We already have `DD_HTTP_SERVER_TAG_QUERY_STRING` to disble the feature, so I'll just remove the empty string comment here. 
Thank you

File: content/en/tracing/trace_collection/library_config/dotnet-core.md [hunk 3]
```diff
@@ -201,6 +201,17 @@ Note that the delimiter is a comma and a space: `, `.
 **Example**: `mysql:main-mysql-db, mongodb:offsite-mongodb-service`<br>
 The `from-key` value is specific to the integration type, and should exclude the application name prefix. For example, to rename `my-application-sql-server` to `main-db`, use `sql-server:main-db`. Added in version 1.23.0
 
+`DD_HTTP_SERVER_TAG_QUERY_STRING`
+: When set to `true`, the `http.url` includes query string parameters. You can find more details in [Redacting the query in the url][14].
+**Default**: `true`
+
+`DD_HTTP_SERVER_TAG_QUERY_STRING_SIZE`
+: When `DD_HTTP_SERVER_TAG_QUERY_STRING` is true, sets the max size of the querystring to report, before obfuscation. Set 0 for no limit in size<br>
```
Comment (rtrieu) [inline content/en/tracing/trace_collection/library_config/dotnet-core.md:209]:
```suggestion
: When `DD_HTTP_SERVER_TAG_QUERY_STRING` is true, the max size of the querystring to report is set, before obfuscation. Set 0 for no limit in size<br>
```

**Review by rtrieu — CHANGES_REQUESTED** [review]
Minor suggestion added, but otherwise looks good!

**Review by rtrieu — APPROVED** [review]
Looks good!

**Pull request merged**